libs/vkd3d/command.c

```diff
@@ -2596,18 +2596,6 @@ static HRESULT STDMETHODCALLTYPE d3d12_command_list_Close(d3d12_command_list_ifa
     if (list->is_predicated)
         VK_CALL(vkCmdEndConditionalRenderingEXT(list->vk_command_buffer));
 
-    // TODO We should emit a barrier between submissions,
-    // not between command lists within the same submission.
-    memory_barrier.sType = VK_STRUCTURE_TYPE_MEMORY_BARRIER;
```

Comment:
The declaration of `memory_barrier` should also be removed.

Reply:
Fixed.